armada/handlers/armada.py

```diff
@@ -102,16 +108,17 @@ def pre_flight_ops(self):
                         try:
                             LOG.info('Cloning repo: %s', location)
                             repo_dir = git.git_clone(location, reference)
-                        except Exception as e:
-                            raise ValueError(e)
+                        except Exception:
```

Comment:
As part of the criteria for this implementation, @gardlt requested that we have exceptions for an invalid git location. If the pygit2 exception is a location error rather an a generic GitError, then I believe it make sense to remove the exception handling and use their exception. Otherwise, the error feedback would not be as helpful.